project/repository.bzl

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Bazel rule for loading external repository deps for j2cl_proto."""
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+_JSINTEROP_BASE_VERSION = "master"
```

Comment:
I think it would be preferable to pin down the version for reproducible builds and avoiding surprise breakages if master introduces backwards incompatible change.

Reply:
Yes, that's important -[version 1.0.0](https://github.com/google/jsinterop-base/releases/tag/1.0.0) works pretty nice, I have changed it ✅